mock-uwv/index.js

```diff
@@ -0,0 +1,40 @@
+var express = require('express')
+var app = express()
+
+app.get('/uwv/bsn/999852147', (req, res) => {
+  res.writeHead(200, { 'Content-Type': 'application/json' });
+  res.write(JSON.stringify({
+    "_embedded": {
+        "gegevensuwv": [
+            {
+                "burgerservicenummer": "999852147",
+                "naam": {
+                    "volledigeNaam": "Tyra Andersson",
+                    "voornamen": "Tyra",
+                    "voorletters": "T",
+                    "geslachtsnaam": "Andersson",
+                    "voorvoegsel": ""
+                },
+                "geboortedatum": "05-11-1977",
+                "aantalUurAfgelopenMaandMeerDanNul": "Ja",
```

Comment:
Same thing goes for this as the BD API, we can make a little bit of a suggestion for a proper API, with respect to yes/no -> boolean, currency and stuff like typeArbeidsovereenkomst, which could likely be an enum-kind-of-thing.

Would be okay with merging without if we have an issue to remind us we might circle back to it.

Reply:
> How can I test this? I used the docker-compose-travis yml, but the bd and uwv options didn't show in the select box

Tested with waardepapieren-component outside docker, not including them in haarlem-frontend is by design as this gives us room to have bd-frontend and uwv-frontend instead. If necessary, we could still include it later. 

I'm good with merging this PR as is:)